build-package.sh

```diff
@@ -577,6 +599,13 @@ for ((i=0; i<${#PACKAGE_LIST[@]}; i++)); do
 		else
 			termux_error_exit "Unknown packaging format '$TERMUX_PACKAGE_FORMAT'."
 		fi
+		# saving a list of compiled packages for further work with it
+		termux_add_package_to_built_packages_list "$TERMUX_PKG_NAME"
+		echo "$TERMUX_BUILD_PACKAGE_CALL_BUILT_PACKAGES_LIST" > $TERMUX_BUILD_PACKAGE_CALL_BUILT_PACKAGES_LIST_FILE_PATH
 		termux_step_finish_build
 	) 5< "$TERMUX_BUILD_LOCK_FILE"
+	if [ -f "$TERMUX_BUILD_PACKAGE_CALL_BUILT_PACKAGES_LIST_FILE_PATH" ]; then
+		export TERMUX_BUILD_PACKAGE_CALL_BUILT_PACKAGES_LIST="$(cat $TERMUX_BUILD_PACKAGE_CALL_BUILT_PACKAGES_LIST_FILE_PATH)"
```

Comment:
No idea what exporting at end of script is gonna do.

Also remove file only if `TERMUX_BUILD_PACKAGE_CALL_DEPTH=0`.